stylus/ui-app/layouts.styl

```diff
@@ -40,20 +40,23 @@ $app-2panes
 
     main,
     main > [role=contentinfo]
-        flex        1
-        height      auto
-        overflow-x  hidden
-        overflow-y  auto
+        display         flex
+        flex-direction  column
+        flex            1 1 auto
+        box-sizing      border-box
+        height          100%
+        overflow-x      hidden
+        overflow-y      auto
 
 // STICKY layout
 // When you want the sidebar to act like a sticky footer
 $app-2panes-sticky
     @extend $app-2panes
 
     @media (max-width: (1023/basefont)rem)
-        height 'calc(100vh - %s)' % em(102px)
+        height 'calc(100vh - %s)' % em(90px, 16px)
         // Occupy all the available height minus the heights of the Cozy-bar
-        // (52px) on top & the navigation (50px) on bottom
+        // (40px) on top & the navigation (50px) on bottom
```

Comment:
I don't like it either. Between that and the z-index, I'm struggling to find something less hazardous. So far i can't find any good idea but I'm open to hear new ones. 
Anyway, until then we can use it as it is, but we clearly should find a better way.